### PR TITLE
Cypress/fe 2003 tile card pages icon

### PR DESCRIPTION
**Pull request by DlgSHi**
# Description
- added tests for Card and Tile to `build.feature`
- updated tests for `Icon` component
- added `pageIndex` test for `Pages` component
- added regression tests for `Tile` / `Card` component

### CODE REVIEW

File: cypress/features/regression/iconTypePart2.feature [hunk 1]
```diff
@@ -59,14 +63,15 @@ Feature: Icon component
...
       | uploaded          |
+      | video             |
       | view              |
       | warning           |
-      | white-tick        |
 
   @positive
   Scenario Outline: Change type to <iconName>
     When I select type to "<iconName>"
     Then icon on preview is "<iconDetaElement>"
```
Comment (klendo) [inline cypress/features/regression/iconTypePart2.feature]: Shouldn't it be iconDataElement?

File: cypress/features/regression/pages.feature [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
+    When I open component preview
       And I close page
     Then My First Page is not visible
       And My Second Page is not visible
 
   @positive
-  Scenario: Open next page
-    When I open no iFrame component preview
-      And I go to next page
+  Scenario: Open Second page
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/pages.feature]: fixed

File: cypress/features/regression/pages.feature [hunk 3]
```diff
@@ -1,39 +1,50 @@
...
-      And I go to next page
-      And I go to previous page
+    When I open component preview
+      And I go to third page
+    Then My Third Page is visible
+    When I go to first page
     Then My First Page is visible
 
+  @positive
+  Scenario Outline: Change Card pageIndex to <pageIndex>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/pages.feature]: fixed

File: cypress/features/regression/pages.feature [hunk 4]
```diff
@@ -1,39 +1,50 @@
...
+    When I open component preview
     Then My First Page is visible
 
   @positive
   Scenario: Open and close first page
-    When I open no iFrame component preview
+    When I open component preview
       And I close page
     Then My First Page is not visible
       And My Second Page is not visible
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/pages.feature]: fixed

File: cypress/features/regression/pages.feature [hunk 5]
```diff
@@ -1,39 +1,50 @@
...
+  Scenario: Open Second page
+    When I open component preview
+      And I go to second page
     Then My Second Page is visible
 
   @positive
-  Scenario: Open and close next page
-    When I open no iFrame component preview
-      And I go to next page
+  Scenario: Open and close second page
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/pages.feature]: fixed

File: cypress/features/regression/pages.feature [hunk 6]
```diff
@@ -1,39 +1,50 @@
...
 
   @positive
   Scenario: Go back to first page
-    When I open no iFrame component preview
-      And I go to next page
-      And I go to previous page
+    When I open component preview
+      And I go to third page
+    Then My Third Page is visible
+    When I go to first page
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/pages.feature]: fixed

File: cypress/features/regression/iconTypePart2.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/iconTypePart2.feature]: fixed

**Pull request merged**